### PR TITLE
CFD tutorial

### PR DESCRIPTION
**Pull request by harishwar017**
## Description
Adding Theory Tutorial for Computational Fluid Dynamics (CFD)

## Type of change

Please check the option that is related to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - In this case, we recommend to discuss your modification on GitHub issues before creating the PR
- [ ] Documentations (modification for documents)

## Checklist

- [ ] My code follows [the style guidelines of this project](https://deepchem.readthedocs.io/en/latest/development_guide/coding.html)
  - [ ] Run `yapf -i <modified file>` and check no errors (**yapf version must be  0.32.0**)
  - [ ] Run `mypy -p deepchem` and check no errors
  - [ ] Run `flake8 <modified file> --count` and check no errors
  - [ ] Run `python -m doctest <modified file>` and check no errors
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New unit tests pass locally with my changes
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by harishwar017** [comment]
@rbharath Please review this PR, I have added tutorial of CFD

File: examples/tutorials/CFD_Theory_Tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,197 @@
+{
```
Comment (rbharath) [inline examples/tutorials/CFD_Theory_Tutorial.ipynb:1]: Please add this on to the CSV files to make sure it gets rendered

Comment (shreyasvinaya) [inline examples/tutorials/CFD_Theory_Tutorial.ipynb:1]: <p>Please fix the rendering, you will have to add a space after the '###'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3958/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (shreyasvinaya) [inline examples/tutorials/CFD_Theory_Tutorial.ipynb:1]: <p>Same rendering issue here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3958/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

File: examples/tutorials/CFD_Theory_Tutorial.ipynb [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+  "nbformat": 4,
```
Comment (rbharath) [inline examples/tutorials/CFD_Theory_Tutorial.ipynb:2]: Check the https://lorenabarba.com/blog/cfd-python-12-steps-to-navier-stokes/ tutorial series for code and examples

File: examples/tutorials/CFD_Theory_Tutorial.ipynb (hunk 1 above)
Comment (rbharath) [reply examples/tutorials/CFD_Theory_Tutorial.ipynb:1]: Check https://github.com/deepchem/deepchem.github.io/ for website rendering of tutorials to make sure it gets added

**Comment by harishwar017** [comment]
I have added the file to Deep Differential Equations CSV

**Pull request closed without merging**